docs/supported-components.human

```diff
@@ -66,6 +66,10 @@ raw-body
   0.0.1-2.4.1
 redis
   0.8.0-3.1.2
+restify
+  4.1.0-6.2.0
```

Comment:
We actually support `8.5.1` which is `latest` (see Accept run).
Anything above `6.2.0` produces an error with `primordials is not defined` but only in `testeachversion` run.